### DISCUSSION TITLE
Jump Top/Bottom

### DISCUSSION THREAD

**Discussion by mhvt02** [gsantner/markor#1697] [Q&A] [answered]
I finally figured out that I can long press the toolbar for jump to top/bottom.  When working on plain text files, I was using the button on the action bar for this.  Is there a reason the same button isn't available for the text action bar for markdown format?

BTW, thank you so much for this app.  I'm still getting the hang of it, but it's my go-to editor. I'm still learning its potential uses.

(Ha – by now you've probably surmised I'm new at this.  I was going to edit to put it it in Q&A category, but I can't quite figure out how to do that.)

**Comment by gsantner** [comment, marked as answer]
You can long-press the "keyboard" button in the bottom toolbar. It's available in ALL formats.

The top toolbar "filename/title" has own action based on filetype, like browsing todos, searching content, ...

**Comment by harshad1** [comment]
Also note that you can drag the scrollbar - personally I find this easier.
